key-concepts/covenants.adoc

```diff
@@ -1,11 +1,174 @@
 = Covenants
 
-A covenant is a set of link:actions.adoc[action] creators and types, a
-reducer (which is a link:smart_contracts.adoc[smart contract], and a
-Redux link:sagas.adoc[saga] that together indicate the functioning of an
-Interbit blockchain. These actions, smart contract, and saga work
-together to fulfill the functionality of a single Interbit blockchain.
-The `rootSaga` in a smart contract is optional.
+[quote,Merriam Webster Dictionary]
+**covenant**: a written agreement or promise, usually under seal,
+between two or more parties, especially for the performance of some
+action.
+
+Interbit blockchain networks are governed by covenants. Each node in the
+blockchain's network agrees to apply the blockchain's covenant.
+
+image:img/covenant-chains.svg["Each node in an Interbit blockchain
+network applies the same covenant", 500, 170, role="zoom center"]
```

Comment:
Each ~node~ blockchain

The blockchain applies the covenant that the nodes agreed to via consensus.

One node may run many blockchains and therefore many covenants. Some nodes may run different chains and therefore different sets of covenants within the blockchain network.